Profiles/cio-Reaction.xml

```diff
@@ -109,32 +109,47 @@
         <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationCondition" />
       </type>
     </element>
-    <element id="AllergyIntolerance.extension:relationHypersensitivity">
+    <element id="AllergyIntolerance.extension:relationCondition.value[x]">
+      <path value="AllergyIntolerance.extension.value[x]" />
+      <short value="RelationCondition" />
+      <definition value="The condition that the healthcare provider has identified as a reaction." />
+      <alias value="RelatieAandoeningOfGesteldheid" />
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-923" />
+        <comment value="RelationCondition" />
+      </mapping>
+    </element>
+    <element id="AllergyIntolerance.extension:RelationHypersensitivityIntolerance">
       <path value="AllergyIntolerance.extension" />
-      <sliceName value="relationHypersensitivity" />
+      <sliceName value="RelationHypersensitivityIntolerance" />
       <type>
         <code value="Extension" />
-        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationHypersensitivity" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationHypersensitivityIntolerance" />
       </type>
     </element>
-    <element id="AllergyIntolerance.extension:relationHypersensitivity.value[x]">
+    <element id="AllergyIntolerance.extension:RelationHypersensitivityIntolerance.value[x]">
       <path value="AllergyIntolerance.extension.value[x]" />
-      <definition value="Relation to the underlying hypersensitivity." />
+      <definition value="Relation to the underlying hypersensitivity intolerance." />
       <mapping>
         <identity value="cio-dataset-200-beta2-20231214" />
         <map value="cio-dataelement-20-823" />
-        <comment value="RelationHypersensitivity" />
+        <comment value="RelationHypersensitivityIntolerance" />
       </mapping>
     </element>
     <element id="AllergyIntolerance.extension:medicationHypersensitivityIdentifier">
       <path value="AllergyIntolerance.extension" />
       <sliceName value="medicationHypersensitivityIdentifier" />
       <type>
         <code value="Extension" />
-        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivityIdentifier" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivity.Identifier" />
       </type>
       <isModifier value="false" />
     </element>
+    <element id="AllergyIntolerance.extension:medicationHypersensitivityIdentifier.url">
```

Comment:
This element is incorrect and should be removed (it still contains the previous url).